python/pyspark/sql/pandas/functions.py

```diff
@@ -43,303 +43,228 @@ class PandasUDFType(object):
 @since(2.3)
 def pandas_udf(f=None, returnType=None, functionType=None):
     """
-    Creates a vectorized user defined function (UDF).
+    Creates a pandas user defined function (a.k.a. vectorized user defined function).
+
+    Pandas UDFs are user defined functions that are executed by Spark using Arrow to transfer
```

Comment:
I don't review it carefully and assume most of it are coped from the md file.

Reply:
Yeah... it happened to be almost a copy ...